From a6906ee560e07383ff4c521167d103382d416cba Mon Sep 17 00:00:00 2001
From: Gianfranco Costamagna <costamagnagianfranco@yahoo.it>
Date: Thu, 7 Sep 2017 09:33:21 +0200
Subject: [PATCH] Fix build with new glibc 2.26  * The nonstandard header
 <xlocale.h> has been removed.  Most programs should    use <locale.h>
 instead.  If you have a specific need for the definition of    locale_t with
 no other declarations, please contact    libc-alpha@sourceware.org and
 explain.

---
 clientgui/AsyncRPC.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/clientgui/AsyncRPC.cpp b/clientgui/AsyncRPC.cpp
index f196ced8b1..e9c249d415 100644
--- a/clientgui/AsyncRPC.cpp
+++ b/clientgui/AsyncRPC.cpp
@@ -19,7 +19,7 @@
 #pragma implementation "AsyncRPC.h"
 #endif
 
-#if !(defined(_WIN32) || (defined(__WXMAC__) && (MAC_OS_X_VERSION_MIN_REQUIRED < MAC_OS_X_VERSION_10_4)))
+#if !(defined(_WIN32) && (defined(__WXMAC__) && (MAC_OS_X_VERSION_MIN_REQUIRED < MAC_OS_X_VERSION_10_4)))
 #include <xlocale.h>
 #endif
 
-- 
2.11.0

